components/time/src/zone/mod.rs

```diff
@@ -229,7 +236,9 @@ pub(crate) mod ule {
         /// from time zone to time zone. The time zone display name of this variant
         /// may or may not be called "Standard Time".
         Standard = 0,
-        /// Indicates the period in a time zone with a higher UTC offset.
+        /// Indicates the period in a time zone with a higher UTC offset than `Standard`.
+        ///
+        /// Not every time zone supports the `Daylight` variant.
```

Comment:
All time zones will successfully format a daylight variant, so they all "support" it in theory, they just don't _use_ it.

```suggestion
        /// Not all time zones use the `Daylight` variant.
```